website/content/docs/k8s/connect/ingress-controllers.mdx

```diff
@@ -0,0 +1,74 @@
+---
+layout: docs
+page_title: Ingress Controller Integrations
+description: Configuring Ingress Controllers With Consul On Kubernetes
+---
+
+# Configuring Ingress Controllers with Consul on Kubernetes
+
+-> This topic requires familiarity with [Ingress Controllers](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/)
+
+-> The following information requires Consul 1.10+, Consul-k8s 0.26+, Consul-helm 0.32+ configured with
+[Transparent Proxy](/docs/connect/transparent-proxy) mode enabled.
+
+This page describes a general approach for integrating Ingress Controllers with Consul on Kubernetes to secure traffic from the Controller
+to the backend services. This allows Consul to transparently manage secure traffic from the ingress point through the entire traffic flow of the service.
```

Comment:
```suggestion
to the backend services. This allows Consul to transparently secure traffic from the ingress point through the entire traffic flow of the service.
```

Reply:
This reads a bit strange to me. I think omitting "manage" flows a bit better, and still gets the point across.